cmd/pulumi-test-language/tests/testdata/l1-resource-keyword-overlap/main.pp

```diff
@@ -0,0 +1,41 @@
+# Keywords in various languages should be renamed and work.
+class = "class_output_string"
+export = "export_output_string"
+import = "import_output_string"
+mod = "mod_output_string"
+object = { object = "object_output_string" }
+self = "self_output_string"
+this = "this_output_string"
+if = "if_output_string"
+
+output "output_class" {
```

Comment:
What if we name these outputs without the `output_` prefix, as in the L2 test? Does that work?